src/main/kotlin/no/nav/familie/oppdrag/domene/OppdragId.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.familie.oppdrag.domene
+
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.trygdeetaten.skjema.oppdrag.Oppdrag
+
+
+data class OppdragId(val fagsystem : String,
+                     val fødselsnummer : String,
+                     val behandlingsId : String)
+
+val Oppdrag.id : OppdragId
+    get() = OppdragId(this.oppdrag110.kodeFagomraade,
+                      this.oppdrag110.oppdragGjelderId,
+                      this.oppdrag110.oppdragsLinje150?.get(0)?.henvisning!!)
```

Comment:
Det er en metode i kontrakten for å hente ut riktig behandlingsid basert på utbetalingsoppdraget (input-formatet). Kan vi bruke den, så sikrer vi at implementasjonen for å hente gjeldende behandlingsid er lik overalt?

Reply:
Ja, det bør vi. I'll fix